lib/tasks/gettext_task_override.rb

```diff
@@ -1,13 +1,47 @@
 module GettextI18nRailsJs
   module Task
+    # use combined .po
     def files_list
       require "gettext_i18n_rails/tasks"
 
       ::Pathname.glob(::File.join(locale_path, 'combined', "**", "*.po"))
     end
 
+    # output to ui-classic, not Rails.root; oldjs, not assets/javascripts
     def output_path
-      ::ManageIQ::UI::Classic::Engine.root.join(GettextI18nRailsJs.config.output_path)
+      ::ManageIQ::UI::Classic::Engine.root.join('app/javascript/oldjs/locale')
+    end
+
+    # use path/lang.json instead of path/lang/app.js
+    def destination(lang)
```

Comment:
Is this method used anywhere?

Reply:
It is.. https://github.com/webhippie/gettext_i18n_rails_js/blob/master/lib/gettext_i18n_rails_js/task.rb#L84

(`po_to_json` calls `files_iterate ; print_footer`)